http-netty/src/main/java/io/micronaut/http/netty/body/NettyJsonHandler.java

```diff
@@ -53,6 +54,7 @@
 @Produces({MediaType.APPLICATION_JSON, MediaType.APPLICATION_HAL_JSON})
 @Consumes({MediaType.APPLICATION_JSON, MediaType.APPLICATION_HAL_JSON})
 @BootstrapContextCompatible
+@Requires(classes=JsonMapper.class)
```

Comment:
probably ok, but might be better as `@Requires(beans=JsonMapper.class)`

Reply:
Changed